src/Umbraco.ModelsBuilder/Dashboard/BuilderDashboardHelper.cs

```diff
@@ -1,21 +1,17 @@
-﻿using System;
-using System.Text;
-using Umbraco.Core.Configuration;
+﻿using System.Text;
+using Umbraco.Core.Composing;
 using Umbraco.ModelsBuilder.Configuration;
 using Umbraco.ModelsBuilder.Umbraco;
 
 namespace Umbraco.ModelsBuilder.Dashboard
 {
     internal static class BuilderDashboardHelper
     {
-        public static bool CanGenerate()
-        {
-            return UmbracoConfig.For.ModelsBuilder().ModelsMode.SupportsExplicitGeneration();
-        }
+        private static Config Config => Current.Configs.ModelsBuilder();
```

Comment:
inject? means making this not static, or passing this in to the methods, also ideally we'd find a better name for this than "Helper"